libs/util/src/makeDiscordCdnUrl.test.ts

```diff
@@ -0,0 +1,37 @@
+import { makeDiscordCdnUrl } from './';
+
+describe('make discord cdn url', () => {
```

Comment:
Should get rid of that describe layer. It was in the other place to clarify which function its coming from.